mix.exs

```diff
@@ -18,6 +19,9 @@ defmodule Nostrum.Mixfile do
     ]
   end
 
+  defp elixirc_paths(:test), do: ["lib", "test"]
```

Comment:
Since we moved the stubs module into the test directory do you think we should get rid of this?

Reply:
I don't think we can get rid of this. Attempting to remove these changes results in the tests not compiling (because the `Stubs` module needs to be compiled). If you have an alternative solution, let me know.